lusSTR/cli/config.py

```diff
@@ -0,0 +1,106 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2020, DHS.
+#
+# This file is part of lusSTR (http://github.com/bioforensics/lusSTR) and is licensed under
+# the BSD license: see LICENSE.txt.
+#
+# This software was prepared for the Department of Homeland Security (DHS) by the Battelle National
+# Biodefense Institute, LLC (BNBI) as part of contract HSHQDC-15-C-00064 to manage and operate the
+# National Biodefense Analysis and Countermeasures Center (NBACC), a Federally Funded Research and
+# Development Center.
+# -------------------------------------------------------------------------------------------------
+
+import argparse
+import lusSTR
+import os
+from pathlib import Path
+from pkg_resources import resource_filename
+import yaml
+
+
+def main(args):
+    Path(args.workdir).mkdir(parents=True, exist_ok=True)
+    final_dest = f"{args.workdir}/config.yaml"
+    config = resource_filename("lusSTR", "data/config.yaml")
+    final_config = edit_config(config, args)
+    with open(final_dest, "w") as file:
+        yaml.dump(final_config, file)
+
```

Comment:
I created the command ```lusstr config``` to create the config file. Settings can be specified as command line arguments. If no arguments are provided, the default settings are used and the user can edit manually. If a working directory is specified, the config file is dumped there, otherwise to the cwd.